<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigatorPage</class>
 <widget class="QWidget" name="NavigatorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
background-color: rgb(115, 115,115);
}</string>
  </property>
  <widget class="QPushButton" name="NButton1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NButton2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>网络</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NButton3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>牵引</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NButton4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>制动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NButton5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>辅助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NButton6">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>空调</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NButton7">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>监控</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NButton8">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>15</y>
     <width>115</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 bold 18px ;
color: rgb(255, 255, 255);
background-color: rgb(51, 153, 255);
border-color: rgb(216, 216, 216);</string>
   </property>
   <property name="text">
    <string>维护</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>NButton1</zorder>
  <zorder>NButton2</zorder>
  <zorder>NButton3</zorder>
  <zorder>NButton4</zorder>
  <zorder>NButton5</zorder>
  <zorder>NButton6</zorder>
  <zorder>NButton7</zorder>
  <zorder>NButton8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
